airbyte-integrations/connectors/source-mixpanel/source_mixpanel/streams/base.py

```diff
@@ -104,48 +105,24 @@ def parse_response(
         # parse the whole response
         yield from self.process_response(response, stream_state=stream_state, **kwargs)
 
-        if self.reqs_per_hour_limit > 0:
-            # we skip this block, if self.reqs_per_hour_limit = 0,
-            # in all other cases wait for X seconds to match API limitations
-            self.logger.info(f"Sleep for {3600 / self.reqs_per_hour_limit} seconds to match API limitations after reading from {self.name}")
-            time.sleep(3600 / self.reqs_per_hour_limit)
+        # if self.reqs_per_hour_limit > 0:
+        #     # we skip this block, if self.reqs_per_hour_limit = 0,
+        #     # in all other cases wait for X seconds to match API limitations
+        #     self.logger.info(f"Sleep for {3600 / self.reqs_per_hour_limit} seconds to match API limitations after reading from {self.name}")
+        #     time.sleep(3600 / self.reqs_per_hour_limit)
 
-    @property
-    def max_retries(self) -> Union[int, None]:
-        # we want to limit the max sleeping time by 2^3 * 60 = 8 minutes
-        return 3
```

Comment:
max_retries should be passed to error_handler as parameter

Reply:
Yes, but it is `5`, while here it is set to `3`.
https://github.com/airbytehq/airbyte/blob/f09a615affc92dfa9b1ebb6c5355f945816e2036/airbyte-cdk/python/airbyte_cdk/sources/streams/http/error_handlers/http_status_error_handler.py#L16-L23

 Do we want to use default value?

Reply:
Yes, there's no necessary reason to use 3 retries instead of the default 5 retries.